epregressions/tk_window.py

```diff
@@ -108,6 +110,10 @@ def __init__(self):
         self.run_period_option.set(ForceRunType.NONE)
         self.reporting_frequency = StringVar()
         self.reporting_frequency.set(ReportingFreq.HOURLY)
+        self.force_output_sql = StringVar()
+        self.force_output_sql.set(ForceOutputSQL.NOFORCE.value)
+        self.force_output_sql_unitconv = StringVar()
+        self.force_output_sql_unitconv.set(ForceOutputSQLUnitConversion.NOFORCE.value)
```

Comment:
StringVars to store it. 

(Maybe it'd be better to just use a command with a callback function. Whatever, I'm following current patterns.)